### ISSUE TITLE
Configurable TTS speed

### ISSUE THREAD

**Issue by dharmab** [dharmab/skyeye#84]
Allow the speaking speed of the TTS to be set via a flag.

**Comment by dharmab** [comment]
We need to fork the piper module to do this but it should be easy.

**Comment by nabbl** [comment]
forked already because of macos build. Will look at this also

**Comment by nabbl** [comment]
https://github.com/nabbl/piper/pull/1

**Comment by dharmab** [comment]
Completed in #147
